modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -7,7 +7,32 @@ class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
         def submit
-          render json: { status: 'OK' }
+          bgs_claim = find_bgs_claim!(claim_id: params[:claimId])
+
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found') if bgs_claim.blank?
+
+          render json: { status: 'OK', success: true }
+        end
+
+        private
+
+        def bgs_service
+          BGS::Services.new(external_uid: target_veteran.participant_id,
+                            external_key: target_veteran.participant_id)
+        end
```

Comment:
I agree with Austin's comment of moving this to the application controller. I think we go ahead and move this one so we don't have to modify it later. We can use the ticket to evaluate v2 and move others.

Reply:
[eb92f10](https://github.com/department-of-veterans-affairs/vets-api/pull/10822/commits/eb92f1054725fd171b879d4554eb6ceba8e2d866)